PRIORITY_NOTES.md

```diff
@@ -0,0 +1,17 @@
+This file contains some notes on the Jenkins job priorities that are configured in this repository.
+
+The ROS buildfarm is currently setup with a Jenkins plugin that manages job priorities.  The priorities are inverse numerically based; that is, a lower number means a higher priority.  As of this writing, the buildfarm is configured to have a range of 1 to 109.  There are periodic maintenance jobs that use priorities 1 to 49, thus those priorities should never be used for distribution jobs.
```

Comment:
Please start each sentence on a new line.

Reply:
Sure, will fix that up.